### DISCUSSION TITLE
Disable Whitelist

### DISCUSSION THREAD

**Discussion by CodyRWhite** [Romanitho/Winget-AutoUpdate#93] [General]
Is there a way to move away from whitelist. I have tried removing the whitelist file and adding an excluded file but all apps fail to update. 

```powershell
18:22:41 - Python 3 : Skipped upgrade because it is not in the included app list
18:22:41 - Microsoft Visual C++ 2013 Redistribut… : Skipped upgrade because it is not in the included app list
```

**Comment by Romanitho** [comment]
Yes. If whitelist was enabled, it is set up in config. So removing the included app list will just tell WAU to update nothing.
To change that, you have 2 options:  
- reinstall WAU without whitelist option
- change whitelist settings (in registry if you use from 1.11.0, see 1.11.0 changelog)
In both cases, dont forget to provide excluded app list to avoid certain apps to autoupdate via WAU (browsers for instance)
